### ISSUE TITLE
[Feature request] add download filename(s) to outputs variable

### ISSUE THREAD

**Issue by vcup** [robinraju/release-downloader#616]
**Is your feature request related to a problem? Please describe.**
when using wildcard '*', i want to get exactly file(s) name

**Describe the solution you'd like**
Add an output variable like downloaded_files to get the filenames that were downloaded.
To access a specific file name, use '${{ fromJson(steps.<step-id>.outputs.downloaded_files)[0] }}'.

**Comment by vcup** [comment]
i'am already forked and working repo, use or test with this
```yaml
- name: download release
  id: dl_rel
  uses: vcup/release-downloader@v1.7
  with:
    repository: user/repo
    latest: true
    fileName: repo-release-*-v*.tbz2
- name: echo files
  run: |
    echo ${{ steps.dl_rel.outputs.downloaded_file_0 }}
    echo ${{ steps.dl_rel.outputs.downloaded_file_1 }}
    ...
    echo ${{ steps.dl_rel.outputs.downloaded_file_n }}
```

**Comment by robinraju** [comment]
Implemented in #628
